src/CreateHandler/CreateTemplate.php

```diff
@@ -101,7 +101,7 @@ public function generateTemplate(
         );
     }
 
-    private function resolveRendererType(?string $templateSuffix): string
+    private function resolveRendererType(?string $templateSuffix): ?string
```

Comment:
Is there a way to maintain a stricter return type here? 🤔

Reply:
https://github.com/mezzio/mezzio-tooling/blob/04adfc1564bd6bdc29d5041e25bcce309b7ef807/src/CreateHandler/CreateTemplate.php#L110
https://github.com/mezzio/mezzio-tooling/blob/04adfc1564bd6bdc29d5041e25bcce309b7ef807/src/TemplateResolutionTrait.php#L87-L94
@Ocramius Can we throw exception if `$type` is null?